### PR TITLE
feat: add best practices workflow

### PR DESCRIPTION
**Pull request by carlossteinzup**
### Related Issues

Update / add opensource security workflows at Beagle Android repository
https://app.shortcut.com/devrelopensource/story/179033/scorecard-beagle-android

### Description and Example

Adds validation actions for:
- DCO, 
- License, 
- Dependabot, 
- Horusec.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.
- [x] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle-android/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
Should call Analytics Service with correct parameters | br.com.zup.beagle.android.view.viewmodel.AnalyticsViewModelTest$CreateActionReport | 0.38|


### Tests: 

Name | Classname | Time|
--- | --- | ---|
Should call Analytics Service with correct parameters | br.com.zup.beagle.android.view.viewmodel.AnalyticsViewModelTest$CreateScreenReport | 0.387|


<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .github/workflows/horusec-pr.yml [hunk 1]
```diff
@@ -44,5 +45,5 @@ jobs:
         env:
```
Comment (hectorcustodiozup) [inline .github/workflows/horusec-pr.yml:45]:
```suggestion
```

File: .github/workflows/horusec-pr.yml [hunk 2]
```diff
@@ -44,5 +45,5 @@ jobs:
...
           HORUSEC_CLI_REPOSITORY_AUTHORIZATION: ${{ env.BEAGLE_CORE_SECURITY_HORUSEC_CLI_REPOSITORY_AUTHORIZATION }}
```
Comment (hectorcustodiozup) [inline .github/workflows/horusec-pr.yml:46]:
```suggestion
```

**Review by hectorcustodiozup — CHANGES_REQUESTED** [review]

**Comment by carlossteinzup** [comment]
Did the modifications. Can you review again please?

**Review by hectorcustodiozup — APPROVED** [review]

**Pull request merged**